### PR TITLE
feat: support maxwell format 

### LINKED ISSUE #6044: connector: support format maxwell-json
https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/formats/maxwell/#:~:text=Maxwell%20Format&text=Maxwell%20is%20a%20CDC%20(Changelog,to%20serialize%20messages%20using%20JSON.

### PR DESCRIPTION
**Pull request by tabVersion**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

***PLEASE DO NOT LEAVE THIS EMPTY !!!***

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:

as title

- Summarize your change (**mandatory**)
- How does this PR work? Need a brief introduction for the changed logic (optional)
- Describe clearly one logical change and avoid lazy messages (optional)
- Describe any limitations of the current code (optional)

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

If your pull request contains user-facing changes, please specify the types of the changes, and create a release note. Otherwise, please feel free to remove this section.

### Types of user-facing changes

Please keep the types that apply to your changes, and remove those that do not apply.

* Connector (sources & sinks)

### Release note

Please create a release note for your changes. In the release note, focus on the impact on users, and mention the environment or conditions where the impact may occur.

allow create source with maxwell format. Maxwell is still a CDC format, pk is required. 

```sql
create materialized source s8 (
  id integer,
  first_name varchar,
  last_name varchar,
  email varchar,
  PRIMARY KEY (id)
) with (
  connector = 'kafka',
  topic = 'xxxx',
  properties.bootstrap.server = '127.0.0.1:29092'
) row format maxwell
```

## Refer to a related PR or issue link (optional)

resolve #6044

### CODE REVIEW

File: src/source/src/parser/maxwell/json.rs [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+        match event.op.as_str() {
+            MAXWELL_INSERT_OP => {
+                let after = event.data.as_ref().ok_or_else(|| {
+                    RwError::from(ProtocolError(
+                        "data is missing for creating event".to_string(),
+                    ))
+                })?;
+                writer.insert(|column| {
+                    json_parse_value(&column.data_type, after.get(&column.name)).map_err(Into::into)
```
Comment (waruto210) [inline src/source/src/parser/maxwell/json.rs:57]: Why maxwell and debezium parser don't use `simd_json_parse_value`.

**Review by waruto210 — APPROVED** [review]
Rest LGTM.

File: src/source/src/parser/maxwell/json.rs (hunk 1 above)
Comment (tabVersion) [reply src/source/src/parser/maxwell/json.rs:57]: Will try with simd-json

**Comment by tabVersion** [comment]
merge first, will add more tests later

**Pull request merged**